practice/pyramid/site.css

```diff
@@ -0,0 +1,38 @@
+
+.row {
+  display: flex;
+  border: 1px solid black;
+  margin: 20px;
+  justify-content: space-between;
+}
+
+.first-row-item {
+  flex: 1;
+  margin: 10px;
+  height: 78px;
+  border: 1px solid black;
+}
+
+.second-row-item1 {
+  flex: 4;
+  margin: 10px;
+  height: 78px;
+  border: 1px solid black;
+}
+
+.second-row-item2 {
+  flex: 1;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 0;
+  margin-right: 10px;
+  height: 78px;
+  border: 1px solid black;
+}
+
+.third-row-item {
+  margin: 10px;
+  height: 78px;
+  width: 90px;
+  border: 1px solid black;
+}
```

Comment:
A trick you can use to get even spacing despite the fact that vertical margins collapse and horizontal ones don't is to use padding. If you apply half the desired "margin" as padding on the parent, then the other half as margin on the children, you'll get equal spacing.

```css
.row {
  padding: 5px;
}

.row > div {
  margin: 5px;
}
```